patcher/src/ts/sidebar/CharacterSelect.tsx

```diff
@@ -87,17 +97,35 @@ class CharacterSelect extends React.Component<CharacterSelectProps, CharacterSel
   }
 
   getSelectedIndex = () : number => {
-    return this.props.characters.findIndex((c: restAPI.SimpleCharacter) => c.id === this.props.selectedCharacter.id);
+    let selectedIndex = this.characters.findIndex((c: restAPI.SimpleCharacter) => c.id === this.selectedCharacter.id);
```

Comment:
const not let, also may as well just return the expression, as you don't do anything with selectedIndex